static/app/components/events/interfaces/frame/context.tsx

```diff
@@ -109,8 +109,9 @@ const Context = ({
     },
     {
       enabled:
-        organization?.features.includes('codecov-stacktrace-integration') &&
-        organization?.codecovAccess &&
+        defined(organization) &&
+        organization.features.includes('codecov-stacktrace-integration') &&
+        organization.codecovAccess &&
```

Comment:
this was the main issue, added the others just to be safe

Reply:
Do we also need `defined(project)` here?